lib/include/openamp/rpmsg_rpc_client_server.h

```diff
@@ -0,0 +1,208 @@
+/*
+ * Copyright (c) 2021, L&T Technology Services Ltd.
+ * All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#ifndef RPMSG_RPC_CLIENT_SERVER_H
+#define RPMSG_RPC_CLIENT_SERVER_H
+
+#include <openamp/open_amp.h>
+
+#if defined __cplusplus
+extern "C" {
+#endif
+
+#define OPENAMP_PACKED_BEGIN
+#define OPENAMP_PACKED_END __attribute__((__packed__))
```

Comment:
Do not define new macro  but reuse exiting one.

Add `#include <metal/compiler.h>` and reuse  `METAL_PACKED_BEGIN`/` METAL_PACKED_END`; as done in https://github.com/OpenAMP/open-amp/blob/master/lib/include/openamp/virtio_ring.h#L39l
the METAL_PACKED_xx macro depend on the compiler